carla_manual_control/setup.py

```diff
@@ -1,13 +1,45 @@
 """
 Setup for carla_manual_control
 """
+import os
+ROS_VERSION = int(os.environ['ROS_VERSION'])
 
-from distutils.core import setup
-from catkin_pkg.python_setup import generate_distutils_setup
+if ROS_VERSION == 1:
+    from distutils.core import setup
+    from catkin_pkg.python_setup import generate_distutils_setup
 
-d = generate_distutils_setup(
-    packages=['carla_manual_control'],
-    package_dir={'': 'src'}
-)
+    d = generate_distutils_setup(
+        packages=['carla_manual_control'],
+        package_dir={'': 'src'}
+    )
 
-setup(**d)
+    setup(**d)
+
+elif ROS_VERSION == 2:
+    from setuptools import setup
+
+    package_name = 'carla_manual_control'
+    setup(
+        name=package_name,
+        version='0.0.0',
+        packages=['src/' + package_name],
+        data_files=[
+            ('share/ament_index/resource_index/packages',
+             ['resource/' + package_name]),
+            ('share/' + package_name, ['package.xml']),
+            ('share/' + package_name + '/config',
+             ['config/settings.yaml', 'config/sensors.json'])
```

Comment:
settings.yaml and sensors.json are carla_ego_vehicle config files, no such files in manual_control :)

Reply:
Setup.py is corrected now :)